.npmignore

```diff
@@ -1,21 +0,0 @@
-# Packages
```

Comment:
Why delete this?

Reply:
I refactored it to use the "files" array in the package.json file